.github/workflows/solana.yml

```diff
@@ -0,0 +1,203 @@
+name: Solana
```

Comment:
should we migrate the `.pre-commit-config.yaml` as well?

Reply:
this is just the minimum set to keep developing - if you want to migrate it, you're definitely welcome to make another PR :)